libs/sdk-react-native/ios/BreezSDKMapper.swift

```diff
@@ -23,7 +23,7 @@ class BreezSDKMapper {
         var list = [AesSuccessActionDataDecrypted]()
         for value in arr {
             if let val = value as? [String: Any?] {
-                try list.append(asAesSuccessActionDataDecrypted(data: val))
+                list.append(try asAesSuccessActionDataDecrypted(data: val))
```

Comment:
Seems like the linter didn't run? But in any case, I'll correct the generator to position the `try` correctly

Reply:
I've added breez/breez-sdk-rn-generator#6 that should resolve the try hoisting issue with swift linting